App/Presentation/Views/Settings/SettingsOptions.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  SettingsOptions.swift
+//  Tumble
+//
+//  Created by Adis Veletanlic on 10/22/24.
+//
+
+import SwiftUI
+
+struct SettingsOptions<T: SettingsOption & Equatable>: View {
+    @Binding var selectedOption: T
+    let allOptions: [T]
+    
+    var body: some View {
+        SettingsList {
+            SettingsListGroup {
+                ForEach(allOptions, id: \.id) { option in
+                    SettingsRadioButton(
+                        option: option,
+                        selectedOption: $selectedOption
+                    )
+                    if !(allOptions.last?.id == option.id) {
```

Comment:
Divider shows up even after the last option
<img width="319" alt="image" src="https://github.com/user-attachments/assets/72216856-5cb8-41f0-9b4b-2d690ed2aa3a">